client/app/app.js

```diff
@@ -72,7 +72,6 @@ angular.module('managerApp', [
   'ng-slide-down',
   'ovh-angular-jsplumb',
   'tmh.dynamicLocale',
-  'ovh-api-services',
```

Comment:
Please refer to: https://github.com/ovh-ux/ovh-manager-cloud/pull/1524/files#diff-fa428ba3145392773209743143c2c38aR55